scripts/superdesk-interfaces/Vocabulary.ts

```diff
@@ -10,20 +10,30 @@ export interface IVocabulary extends IBaseRestApiResponse {
     helper_text?: string;
     popup_width?: number;
     type: string;
-    items: Array<{name: string; qcode: string}>;
+    items: Array<{ name: string; qcode: string; is_active: boolean }>;
     single_value?: boolean;
     schema_field?: string;
     dependent?: boolean;
     service: {};
     priority?: number;
     unique_field: string;
     schema: {};
-    field_type: 'text' | 'media' | 'date' | 'embed' | 'related_content' | 'custom';
-    field_options?: {};
+    field_type:
+        | 'text'
+        | 'media'
+        | 'date'
+        | 'embed'
+        | 'related_content'
+        | 'custom';
+    field_options?: {
```

Comment:
Is `field_options` specific to related items field? If so, could you rename it to `related_items_options`? We'll probably move it to be a custom field later, but until we do I think it's best to document that it's specific to related items so no other properties get added there which don't belong to related items field.

Reply:
It seems it is, but I can see it as a generic type for links between items. Maybe we could do some renaming. I can add now a comment that's only being used for related items